build.py

```diff
@@ -631,16 +631,25 @@ def onnxruntime_cmake_args(images, library_paths):
                                  'TRITON_ENABLE_ONNXRUNTIME_OPENVINO', False)
         ]
     else:
+        ort_image = None
+        if ('base' in images) and ('ort-base' in images):
+            ort_image = images['ort-base']
+        elif ('base' in images):
+            # If ort-base is not specified, we default to
+            # tensorrt:22.12-py3 base container for ort backend
+            # build with cuda 11.8 library as ORT does
+            # not support cuda 12 in 23.01 containers.
+            ort_image = 'nvcr.io/nvidia/tensorrt:22.12-py3'
```

Comment:
It is strange that we need tensorrt:22.12 when we built with the 22.12 min container in our CI. Do you happen to have the error log that you saw when the min container was used?

Reply:
I think I have lost it. I am trying again with the min container.

Reply:
tritonserver:22.12-py3-min is working. I might not have set the containers properly before.